v2.0/information-schema.md

```diff
@@ -30,6 +30,17 @@ Views | [tables](#tables), [views](#views)| [`SHOW CREATE VIEW`](show-create-vie
 
 The `information_schema` database is comprised of many views representing your cluster's schema, each of which is detailed below.
 
+### applicable_roles
+
+The `applicable_roles` view identifies all roles whose privileges the current user can use. This implies there is a chain
+of role grants from the current user to the role in question. The current user itself is also an applicable role.
```

Comment:
the current user isn't actually included in the view. I [pinged](https://www.postgresql.org/message-id/151747128215.1240.1598372623554241748%40wrigleys.postgresql.org) the postgres docs mailing list about this to figure out if it was odd wording or mismatched docs, no answer. I'm taking their docs to mean: "the listed roles are applicable role; the current user is also an applicable role, but is not listed"

Reply:
Done.